### PR TITLE
Remove test helper from isString function tests

### PR DESCRIPTION
**Pull request by khamiltonuk**
Make codebase more accessible remove testhelper

### CODE REVIEW

**Comment by mroderick** [comment]
I think I made a mistake in #55, and didn't add tests for `referee.refute.isArrayBuffer`.

I think that we should have the `refute` tests for an assertion in the same file as the `assert` tests. This makes it easier to quickly see if there are tests missing. I'll be making a follow up to #55.

Would you mind adding refute tests to this PR?

File: lib/assertions/is-string.test.js [hunk 1]
```diff
@@ -1,63 +1,54 @@
...
+describe("isString", function() {
+    it("should pass for string", function() {
+        referee.assert.isString("Hey");
+    });
+
+    it("should fail for object", function() {
+        var actual;
+        try {
+            referee.assert.isString({});
+        } catch (err) {
```
Comment (mantoni) [inline lib/assertions/is-string.test.js]: You could use [assert.throws](https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message) here.

Comment (mroderick) [reply lib/assertions/is-string.test.js]: We were trying to not use `referee` to test `referee` :)

Comment (mantoni) [reply lib/assertions/is-string.test.js]: In referee that would be `assert.exception`. In node assert it's `assert.throws` 😉

**Comment by mroderick** [comment]
@khamiltonuk would you mind re-visiting this and your other PR, so we can get the changes merged in?

**Comment by khamiltonuk** [comment]
@mroderick thanks for the poke, does this look ok?

**Comment by mroderick** [comment]
Looks great! Thank you 🏆

**Pull request merged**